src/core/MeganavSearchSuggestions/component.html.erb

```diff
@@ -12,7 +12,7 @@
       <%= link_to 'Publish/Subscribe Messaging', abs_url("/docs/core-features/pubsub"), class: "ui-text-p2 hover:text-gui-hover active:text-gui-active focus:text-gui-focus" %>
     </li>
     <li class="py-12 pl-8 flex-shrink-0">
-      <%= link_to 'Platform', abs_url("/docs/how-ably-works"), class: "ui-text-p2 hover:text-gui-hover active:text-gui-active focus:text-gui-focus" %>
+      <%= link_to 'Platform', abs_url("/platform"), class: "ui-text-p2 hover:text-gui-hover active:text-gui-active focus:text-gui-focus" %>
```

Comment:
This needs to updated on the jsx version too

Reply:
@dpiatek it was already changed before here https://github.com/ably/ably-ui/blob/465d57ad98930498c88c999d8fac75d4191563a0/src/core/MeganavSearchSuggestions/component.jsx#L29 

so I did not changed this time